examples/Electrostatic2D3V/README.md

```diff
@@ -0,0 +1,29 @@
+# The Electrostatic2D3V solver
+
+## Overview
+
+The theory behind the Electrostatic2D3V solver and the `two_stream` example can be found in NEPTUNE report [M4c.1: 1-D and 2-D Particle Models](https://excalibur-neptune.github.io/Documents/CD-EXCALIBUR-FMS0070-1.00-M4c.1_ExcaliburFusionModellingSystem.html); see section 2.2 in particular.
+
+## Examples
+
+### two_stream
+
+The example can be run with
+
+    ./scripts/run_eg.sh Electrostatic2D3V two_stream
+
+#### Outputs and postprocessing
+
+Unlike other NESO examples, the Electrostatic2D3V solver doesn't generate Nektar++ checkpoint files.
+Instead it outputs `Electrostatic2D3V_field_trajectory.h5` which contains particle data in [H5Part](https://github.com/zoziha/h5part) format, a derivative of HDF5.
```

Comment:
I suspect that might be a random h5part repo. I used these references:
.. [H5PART] H5Part: A Portable High Performance Parallel Data Interface for Particle Simulations, doi: 10.1109/PAC.2005.1591740. `IEEE <https://ieeexplore.ieee.org/document/1591740>`_ `CERN <https://accelconf.web.cern.ch/p05/papers/fpat083.pdf>`_.

Reply:
Thanks - changed